src/shared/components/VersionInfoOSS.tsx

```diff
@@ -0,0 +1,50 @@
+// Libraries
+import React, {FC, useEffect} from 'react'
+import {useSelector, useDispatch} from 'react-redux'
+
+// Constants
+import {GIT_SHA} from 'src/shared/constants'
+
+// Thunks
+import {fetchVersionInfo} from 'src/shared/thunks/app'
+
+// Selectors
+import {getVersionInfo} from 'src/shared/selectors/app'
+
+const VersionInfoOSS: FC = () => {
+  const dispatch = useDispatch()
+
+  const versionInfo = useSelector(getVersionInfo)
+
+  useEffect(() => {
+    if (!versionInfo.version || !versionInfo.commit) {
```

Comment:
This feels a bit clunky but it was the easiest way I could come up with to prevent the UI from hitting the `/health` endpoint again from the login page when you get to the getting started page. An alternative might be to dispatch the action in some higher-up container but I didn't want to mess with that for this simple change. Open to suggestions!